ledger/metrics/src/test/lib/scala/com/daml/metrics/api/testing/MetricValues.scala

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.api.testing
+
+import com.codahale.metrics.Snapshot
+import com.daml.metrics.api.MetricHandle.{Counter, Histogram, Meter, Timer}
+import com.daml.metrics.api.MetricsContext
+import com.daml.metrics.api.dropwizard.{
+  DropwizardCounter,
+  DropwizardHistogram,
+  DropwizardMeter,
+  DropwizardTimer,
+}
+
+trait MetricValues {
+
+  import scala.language.implicitConversions
+
+  implicit def convertCounterToValuable(counter: Counter): CounterValues = new CounterValues(
```

Comment:
Do you possibly have a reference you could share where this naming convention is described (i.e., `valueable` suffix means you can call `.value` on it), please?

Reply:
Same exact way that scalatest does it, eg: https://github.com/scalatest/scalatest/blob/main/jvm/core/src/main/scala/org/scalatest/OptionValues.scala